go/teams/service_helper.go

```
@@ -545,6 +545,8 @@ func RemoveMember(ctx context.Context, g *libkb.GlobalContext, teamname, usernam
 	if err != nil {
 		if err == errInviteRequired {
 			inviteRequired = true
+		} else if err == errUserDeleted {
+			// no op
```

Post a Debug log and set `err` equal to `nil` if you plan to ignore the error.

---

Also, use a `switch`